Example/SSDynamicTextTests/SSTextViewTests.m

```diff
@@ -0,0 +1,106 @@
+//
+//  SSTextViewTests.m
+//  SSDynamicTextExample
+//
+//  Created by Remigiusz Herba on 31/08/15.
+//  Copyright (c) 2015 Splinesoft. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+#import <SSDynamicTextView.h>
+#import <OCMock/OCMock.h>
+#import "SSDynamicsView.h"
+#import "SSAttributedStringValidator.h"
+
+static NSString * const kTestFontName = @"Avenir-Roman";
+static CGFloat    const kTestFontSize = 14.f;
+static CGFloat    const kTestFontSizeDifferenceForSizeEELarge = 2.f;
+
+@interface SSTextViewTests : XCTestCase
+
+@property (nonatomic, strong) SSDynamicTextView *dynamicTextView;
+@property (nonatomic, strong) id applicationMock;
+@property (nonatomic, strong) SSDynamicTextView *dynamicTextViewFromXib;
+
+@end
+
+@implementation SSTextViewTests
+
+- (void)setUp {
+    [super setUp];
+    self.applicationMock = OCMPartialMock([UIApplication sharedApplication]);
+    self.dynamicTextView = [SSDynamicTextView textViewWithFont:kTestFontName baseSize:kTestFontSize];
+
+    SSDynamicsView *view = [[NSBundle mainBundle] loadNibNamed:@"SSDynamicsView" owner:nil options:nil].firstObject;
+    self.dynamicTextViewFromXib = view.textView;
+}
+
+- (void)tearDown {
+    [self.applicationMock stopMocking];
+    [super tearDown];
+}
+
+- (void)mockEELargeCategory {
```

Comment:
Every single test is using:
 `-mockEELargeCategory`
and 
`[[NSNotificationCenter defaultCenter] postNotificationName:UIContentSizeCategoryDidChangeNotification object:nil];`
So you could extract this to some helper, but __up to you__.